### ISSUE TITLE
Probably an XP booster thing?

### ISSUE THREAD

**Issue by bundillion** [JohN100x1/IsekaiMod#190]
**Describe the bug**
In Blackwater particularly, as enemies have a specific resistance to overcome in order for them to "die" properly, it's something of an automatic habit to pile on the full attacks while they're down.

This is giving me on-kill exp per hit. Which I can only guess is either a system bug, or something to do with having started the game with the power leveling OP ability.

**To Reproduce**
Steps to reproduce the behavior (assuming it does? electricity also isn't working on them properly):
1. Have power leveling
2. As Isekai Hero (player legacy/magus sword saint/fighter 2h/rogue)
3. Have ascendant element (electricity)
4. Down enemies with damage (zippy magic + chain lightning is stronk)
5. continue full-attacks

**Expected behavior**
Pretty sure this shouldn't happen (but also this is a whole new kind of power-leveling)

**Screenshots**
![image](https://github.com/JohN100x1/IsekaiMod/assets/125714650/a7016807-4180-43e8-beb6-a038de8332b9)
![image](https://github.com/JohN100x1/IsekaiMod/assets/125714650/b1a27805-a8a0-4bc1-93c1-f371591b52c7)

**Additional context**
As far as I remember, electricity damage is supposed to kill their regen as well, and it's not, so there might be (probably is) something else going on under the hood. It's not really a _problem_ so much as it is a bit silly to gain 20-30x the normal exp for a given mob. One of the ones in the same room took 2k damage before collapsing, and I haven't killed him yet.
![image](https://github.com/JohN100x1/IsekaiMod/assets/125714650/cb169ccd-0768-4e50-983f-98dd5f1192c5)
(I should have brought more adamantine)

**Comment by casma164** [comment]
I had this happen to myself also, not in Blackwater but Wintersun.  Just out of curiosity, are you using other mods?

**Comment by bundillion** [comment]
> are you using other mods?

Yes, but this is the first time I've noticed something like this happening (it's also the first time I've used the power leveling ability at all)


> The only similarity is I gave myself the power leveling power to a non-isekai character.

I have it on my MC, but I don't know if giving it to NPC toons would make it worse or better (do they stack? _should_ they stack??)

**Comment by casma164** [comment]
Well, I mean I gave it to my main that did not have any levels in isekai.  Its probably a weird conflict with another mod.
